packages/@aws-cdk-testing/framework-integ/test/aws-s3/test/integ.bucket-legacy-naming.ts

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as iam from 'aws-cdk-lib/aws-iam';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-s3-legacy-name-integ');
+
+const legacyBucketFromName = s3.Bucket.fromBucketName(stack, 'LegacyBucketFromName', 'my_legacy_bucket1');
+
+const legacyBucketFromArn = s3.Bucket.fromBucketArn(stack, 'LegacyBucketFromArn', 'arn:aws:s3:::my_legacy_bucket2');
+
+const legacyBucketFromAttributes = s3.Bucket.fromBucketAttributes(stack, 'LegacyBucketFromAttributes', {
+  bucketName: 'my_legacy_bucket3',
+});
```

Comment:
Was this integ test successfully ran? I'm worried that if we need to update the test and re-deploy, it would fail to deploy due to bucket not found.

Reply:
It got deployed, but did not really check if it can work or not. Any way let me double check it.